client/landing/gutenboarding/available-designs.ts

```diff
@@ -35,6 +35,7 @@ const availableDesigns: Readonly< AvailableDesigns > = {
 				base: 'Raleway',
 			},
 			categories: [ 'featured', 'portfolio' ],
+			premium: true,
```

Comment:
We can ship without marking any of the designs as premium and add this later on.